next.config.js

```diff
@@ -63,6 +93,31 @@ module.exports = nextTranslate({
         source: '/data/api/:slug*',
         destination: 'http://51.89.227.200:7011/data/api/:slug*',
       },
+      {
```

Comment:
This should not be needed anymore after https://github.com/ambanum/OpenTermsArchive.org/pull/191 as you can use a `permalink` directly in the markdown file